app/jobs/get_suggestions_job.rb

```diff
@@ -0,0 +1,93 @@
+class GetSuggestionsJob < ApplicationJob
+  queue_as :default
+
+  def perform(job_status)
+    @job_id = job_status.job_id
+
+    send_spotify_request
+
+    if get_spotify_tracks
+      job_status.update(job_attributes_success)
+    else
+      job_status.update(job_attributes_failure)
+    end
+    job_status.done = true
+    job_status.save!
+  end
+
+  private
+
+  def client_token
+    Base64.strict_encode64(
+      "#{ENV['SPOTIFY_CLIENT_ID']}:#{ENV['SPOTIFY_CLIENT_SECRET']}",
+    ).delete("\n")
+  end
+
+  def send_spotify_request
+    token_url = "https://accounts.spotify.com/api/token"
+    grant_type = { grant_type: "client_credentials" }
+    authorization_basic = { "Authorization": "Basic #{client_token}" }
+
+    @spotify_token = JSON.parse(
+      RestClient.post(token_url, grant_type, authorization_basic),
+    )
+  end
+
+  def get_spotify_tracks
+    track_id = random_track
+    track_url = "https://api.spotify.com/v1/tracks/#{track_id}"
+    access_token = @spotify_token.values[0]
+    authorization_bearer = { "Authorization": "Bearer #{access_token}" }
+
+    @track = JSON.parse(RestClient.get(track_url, authorization_bearer))
+  end
+
+  def random_track
+    breathe_the_fire = "7qLDBd5iCh9xilyfdkn9v8"
```

Comment:
I totally understand why you are naming these things out. Would this be better served in a hash called `random_tracks`?

Something like 
```ruby
random_tracks = {
  breathe_the_fire: "foo",
  # etc
}

random_tracks.to_a.sample
```

Oooh better yet, what about a CONSTANT that is a hash of those things so `random_track` can be super tiny?